classes/doi/Repository.php

```diff
@@ -260,6 +260,20 @@ public function markRegistered(int $doiId)
         $this->edit($doi, $editParams);
     }
 
+    /**
+     * Manually sets DOI status to Doi::STATUS_UNREGISTERED.
+     */
+    public function markUnregistered(int $doiId)
+    {
+        $doi = $this->get($doiId);
+        $editParams = [
+            'status' => Doi::STATUS_UNREGISTERED,
+        ];
+
+        HookRegistry::call('Doi::markUnregistered', [&$editParams]);
```

Comment:
I don't think we need the hook here or in the other bulk actions. It's hard to imagine a use case for them. But if you have one in mind and want to keep the hook, you'll also need to pass the id to it.